apps/package.json

```diff
@@ -26,12 +26,15 @@
     "build:stats": "node ./script/build-stats.js",
     "prestorybook": "curl -o build/package/css/application.css http://localhost:3000/assets/application.css",
     "storybook": "start-storybook -p 9001 -s build/package/",
-    "storybook:deploy": "./script/deploy-storybook.sh"
+    "storybook:deploy": "./script/deploy-storybook.sh",
+    "codemod:es6-modules": "./script/codemod-es6-modules.sh",
+    "codemod:move-file": "./script/codemod-move-file.sh"
```

Comment:
I agree we should upgrade to a more recent version of node.

It is where people go to discover scripts. I guess I'm questioning a little bit whether this will be very widely used. If this list is full of a bunch of scripts people don't need/use, it makes it harder to find the ones that they do.

We've also just added three new packages to our dependency list for a couple of scripts that I would argue are not part of our standard build process/workflow.

This isn't something I feel super strongly about, so you're welcome to ignore me.